### DISCUSSION TITLE
Error downloading input data

### DISCUSSION THREAD

**Discussion by pgoddard08** [E3SM-Project/E3SM#6555] [E3SM model help]
Hello, 

I'm receiving an error regarding the download of input files. I've not had this issue for this compset+grid combination in the past, but during the last couple days of trying this run I've received this error:
```
Loading input file list: 'Buildconf/mpassi.input_data_list'
Model mpassi missing file graph576 = '/N/project/obrienta_startup/datasets/CESMinputfiles/ice/mpas-seaice/IcoswISC30E3r5/partitions/mpas-seaice.graph.info.20231120.part.576'
Trying to download file: 'ice/mpas-seaice/IcoswISC30E3r5/partitions/mpas-seaice.graph.info.20231120.part.576' to path '/N/project/obrienta_startup/datasets/CESMinputfiles/ice/mpas-seaice/IcoswISC30E3r5/partitions/mpas-seaice.graph.info.20231120.part.576' using SVN protocol.
svn export failed with output:  and errput svn: E170000: URL 'https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/ice/mpas-seaice/IcoswISC30E3r5/partitions/mpas-seaice.graph.info.20231120.part.576' doesn't exist

```

Thanks for you help,
Paul

**Comment by jonbob** [comment]
Hi Paul,

That file should get downloaded via wget from the e3sm inputdata repo -- it won't be available from the ccsm svn server. Are you getting any error from wget?

Thanks,
Jon

PS I have verified that it exists, so it's a matter of downloading and not a missing file

**Comment by rljacob** [comment]
We've had some issues with the web.lcrc.anl.gov server.   Try it again.

**Reply by pgoddard08** [reply]
Hi Robert,

I still got the same error as of ~1PM ET

**Reply by rljacob** [reply]
Also still seeing errors.  I'll elevate this with the web site support people.

**Reply by pgoddard08** [reply]
Thanks!

**Reply by rljacob** [reply]
What machine are you issuing the wget command on?

**Reply by pgoddard08** [reply]
I ported E3SMv2.1 and v3 to Indiana University's cluster and am running on that machine (called [Big Red 200](https://kb.iu.edu/d/brcc))

**Reply by rljacob** [reply]
Try now, please.

**Reply by pgoddard08** [reply]
Works! - it used the wget protocol.  Thank you very much.
